doc/source/deployment/requirements.rst

```diff
@@ -105,5 +101,7 @@ Network Requirements
     the Deployer with dnsmasq to supply DNS functionality, but this is beyond
     the scope of this preview.
 
+Distributed Virtual Routing (DVR) is not supported in this Technology Preview.
```

Comment:
@csymons-suse T.T. asked for a section to state what openstack services are supported, and what features are not supported including dvr. so can we add a section in the overview?

Reply:
addressed in separate PR. so I am good here

Reply:
I added to the home/discaimer page. So all good here